src/containers/GameContainer/GameContainer.tsx

```diff
@@ -16,13 +19,48 @@ const config = {
   HERO_DEFAULT_RADIUS: 20,
   HERO_DEFAULT_BOOST: 0.00009,
   HERO_DEFAULT_DEBOOST: 0.00002,
+  TOTAL_TIME: 10000,
 };
+const mainHero = new MainHero({
+  x: config.HERO_START_POSITION_X,
+  y: config.HERO_START_POSITION_Y,
+  radius: config.HERO_DEFAULT_RADIUS,
+  boost: config.HERO_DEFAULT_BOOST,
+  deboost: config.HERO_DEFAULT_DEBOOST,
+  mainStyle: {
+    stroke: config.HERO_BORDER_MAIN_COLOR,
+    fill: config.HERO_FILL_MAIN_COLOR,
+  },
+  loseStyle: {
+    stroke: config.HERO_BORDER_LOSE_COLOR,
+    fill: config.HERO_FILL_LOSE_COLOR,
+  },
+});
+const timeController = new TimeController({
```

Comment:
🧐
Тут и везде лучше отбить пустой строкой, а то сейчас в одну котлету слепилось
```suggestion
});

const timeController = new TimeController({
```

Reply:
Линтер склеивает